x/metadata/keeper/msg_server_test.go

```diff
@@ -202,7 +1222,7 @@ func (s *MsgServerTestSuite) TestAddAndDeleteScopeDataAccess() {
 		_, errDel := s.msgServer.DeleteScopeDataAccess(s.ctx, msgDel)
 		require.NoError(t, errDel, "Failed to make DeleteScopeDataAccessRequest call")
 
-		scopeB, foundB := s.app.MetadataKeeper.GetScope(s.ctx, scopeA.ScopeId)
+		scopeB, foundB := s.app.MetadataKeeper.GetScopeWithValueOwner(s.ctx, scopeA.ScopeId)
 		require.Truef(t, foundB, "Scope %s not found after DeleteScopeOwnerRequest call.", scopeA.ScopeId)
 
 		assert.Equal(t, scopeA.ScopeId, scopeB.ScopeId, "del ScopeId")
```

Comment:
Each test case is run using `s.T().Run` instead of `s.Run`. That means that using `s.Require()` would apply the check to the wrong `t`. So `require` is correct here.